saleor/core/management/commands/cleardb.py

```diff
@@ -0,0 +1,76 @@
+"""Clear random data.
+
+This command removes random data generated by `populatedb` command, such as example
+orders, products or customer accounts. It doesn't remove shop's configuration, such as
+an admin account, site settings or menus.
+"""
+
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+from django.db.models import Q
+
+from saleor.account.models import User
+from saleor.checkout.models import Checkout
+from saleor.discount.models import Sale, Voucher
+from saleor.order.models import Order
+from saleor.payment.models import Payment, Transaction
+from saleor.product.models import Product
+from saleor.shipping.models import ShippingMethod
+
+
+class Command(BaseCommand):
```

Comment:
How about delete:
1. ServiceAccounts
1. GiftCards
1. Categories
1. Collections
1. ProductTypes
1. Attributes
1. ShippingZones
1. Warehouses
1. Webhooks

Reply:
The idea behind the original command that I moved from `demo` branch was to remove data that people create in our demo instance - checkouts, orders, payments, changes in product stock. Then you can easily run the command periodically and repopulate database with fresh data.

Reply:
In that case, we shouldn't close #4971

Reply:
After speaking with @NyanKiyoshi I'll fix this command to remove other models as well, except for service accounts, staff users, site settings and plugins configuration.